### PR TITLE
Add python api for lstmp operator

### LINKED ISSUE #7813: Add Python API for LSTMP operator
For #7792

### PR DESCRIPTION
**Pull request by kuke**
Resolve #7813

### CODE REVIEW

File: python/paddle/v2/fluid/layers/nn.py [hunk 1]
```diff
@@ -389,6 +394,185 @@ def dynamic_lstm(input,
...
+
+        o_t & = \sigma(W_{ox}x_{t} + W_{or}r_{t-1} + W_{oc}c_t + b_o)
+
+        c_t & = f_t \odot c_{t-1} + i_t \odot \\tilde{c_t}
+
+        h_t & = o_t \odot act_h(c_t)
+
+        r_t & = \overline{act_h}(W_{rh}h_t)
+
+    where the :math:`W` terms denote weight matrices (e.g. :math:`W_{xi}` is
```
Comment (pkuyym) [inline python/paddle/v2/fluid/layers/nn.py]: Please reorganize these description like: 
 
* :math:`W_{fc}`: {description}
* :math:`W_{oc}`: {description}

File: python/paddle/v2/fluid/layers/nn.py [hunk 2]
```diff
@@ -389,6 +394,185 @@ def dynamic_lstm(input,
...
+
+    Args:
+        input(Variable): The input of dynamic_lstmp layer, which supports
+                         variable-time length input sequence. The underlying
+                         tensor in this Variable is a matrix with shape
+                         (T X 4D), where T is the total time steps in this
+                         mini-batch, D is the hidden size.
+        size(int): 4 * hidden size.
+        proj_size(int): The size of projection output.
+        param_attr(ParamAttr|None): The parameter attribute for the learnable
```
Comment (pkuyym) [inline python/paddle/v2/fluid/layers/nn.py]: Please add some attribution examples.

File: python/paddle/v2/fluid/layers/nn.py (hunk 1 above)
Comment (kuke) [reply python/paddle/v2/fluid/layers/nn.py]: Done

File: python/paddle/v2/fluid/layers/nn.py (hunk 2 above)
Comment (kuke) [reply python/paddle/v2/fluid/layers/nn.py]: Done

**Pull request merged**